addons/webhooks/clusterbootstrap_webhook_test.go

```diff
@@ -120,11 +120,15 @@ var _ = Describe("ClusterbootstrapWebhook", func() {
 			// CSI should not be touched
 			Expect(clusterBootstrap.Spec.CSI.RefName).To(Equal(fmt.Sprintf("%s.%s", fakeCSICarvelPackageRefName, fakeCarvelPackageVersion)))
 			Expect(clusterBootstrap.Spec.CSI.ValuesFrom.Inline["foo"]).To(Equal("bar"))
-			// AdditionalPackages should not be touched
+			// Existing additionalPackages should not be touched, the ones in ClusterBootstrapTemplate will be added
 			Expect(clusterBootstrap.Spec.AdditionalPackages).NotTo(BeNil())
-			Expect(len(clusterBootstrap.Spec.AdditionalPackages)).To(Equal(1))
-			Expect(clusterBootstrap.Spec.AdditionalPackages[0].RefName).To(Equal(additionalCBPackageRefName))
-			Expect(clusterBootstrap.Spec.AdditionalPackages[0].ValuesFrom.Inline["identity_management_type"]).To(Equal("ldap"))
+			Expect(len(clusterBootstrap.Spec.AdditionalPackages)).To(Equal(len(clusterBootstrapTemplate.Spec.AdditionalPackages)))
```

Comment:
Question: In the case of this example, ```ClusterBootstrapTemplate``` had the same number of entries as ``` ClusterBootstrap ```?

Reply:
I will change it to `2` to make it readable and understandable.